package.json

```diff
@@ -41,47 +42,42 @@
     "@oclif/config": "^1.17.1",
     "@oclif/errors": "^1.3",
     "@oclif/plugin-help": "^3.2.7",
-    "@typescript/vfs": "^1.3.5",
-    "async": "^3.2.0",
+    "@typescript/vfs": "^1.5.0",
     "case": "^1.6.3",
     "chokidar": "^3.5.1",
-    "fs-extra": "^9.1.0",
+    "fs-extra": "^11.1.1",
     "inquirer": "^8.2.0",
     "lodash": "^4.17.21",
-    "lz-string": "^1.4.4",
     "ora": "^5.4.0",
     "prettier": "3.0.3",
     "rxjs": "^7.4.0",
     "slash": "^3.0.0",
     "threads": "^1.7.0",
     "tslib": "^2.3.1",
     "tsutils": "^3.21.0",
-    "typescript": "^4.5.2",
-    "zod": "^3.21.0"
+    "typescript": "^5.2.2",
+    "zod": "^3.22.4"
   },
   "devDependencies": {
-    "@babel/core": "^7.17.5",
-    "@babel/preset-env": "^7.16.11",
-    "@babel/preset-typescript": "^7.16.7",
-    "@types/async": "^3.2.6",
-    "@types/fs-extra": "^9.0.9",
+    "@babel/core": "^7.23.2",
+    "@babel/preset-env": "^7.23.2",
+    "@babel/preset-typescript": "^7.23.2",
+    "@types/fs-extra": "^11.0.3",
     "@types/inquirer": "^8.1.3",
     "@types/jest": "^29.5.7",
     "@types/lodash": "^4.14.168",
-    "@types/lz-string": "^1.3.34",
     "@types/node": "^12.7.11",
-    "@typescript-eslint/eslint-plugin": "^4.17.0",
-    "@typescript-eslint/parser": "^4.17.0",
-    "auto-changelog": "^2.2.1",
+    "@typescript-eslint/eslint-plugin": "^6.9.1",
+    "@typescript-eslint/parser": "^6.9.1",
+    "auto-changelog": "^2.4.0",
     "babel-jest": "^29.7.0",
     "codecov": "^3.8.3",
-    "eslint": "^8.51.0",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-plugin-prettier": "^3.3.1",
-    "husky": "^5.2.0",
+    "eslint": "^8.53.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-prettier": "^5.0.1",
+    "husky": "^8.0.3",
     "jest": "^29.7.0",
-    "pretty-quick": "^3.1.0",
```

Comment:
Any problem with `pretty-quick` ? I usually use this package to lint only what's staged (quicker and better dx)

Reply:
i thought there was an issue caused by a transitive dependency, but it was not the real issue. so I reverted this change